### ISSUE TITLE
Complete rule specification

### ISSUE THREAD

**Issue by Razican** [SUPERAndroidAnalyzer/super#37]
We need to create a complete rule specification, and publish it as an RFC. We have to include current rule specification and decide if we require more fields, or if we want to modify current ones.

**Comment by Razican** [comment]
If the RFC process gets accepted, I will close this issue in favor of an RFC.

**Comment by Razican** [comment]
Droped in favor of a complex rule system, after we ditch out dependencies. On hold.

**Comment by Razican** [comment]
This depends on #22, or at least in #24, so it won't be finished for 0.3.0.
